conductr_cli/conduct_main.py

```diff
@@ -328,6 +328,7 @@ def run(_args=[]):
     parser = build_parser(dcos_mode)
     argcomplete.autocomplete(parser)
     args = parser.parse_args(_args[1:])
+    args.dcos_mode = dcos_mode
```

Comment:
I think it would make sense to assign the values directly to `args.dcos_mode` instead of using an intermediate variable: https://github.com/fsat/conductr-cli/blob/78078ef7d491cd8de8629268cfc44ee9ed478815/conductr_cli/conduct_main.py#L322-L326

```
    if _args and Path(_args[0]).name == constants.DCOS_COMMAND_PREFIX + 'conduct':
        _args = _args[1:]
        args.dcos_mode = True
    else:
        args.dcos_mode = False
```

Then we don't need to do:

```
args.dcos_mode = dcos_mode
```

This is also consisten with the way we are doing it for other variables. We set args.x always directly.